templates/patterns/featured_item/featured_item.ui_patterns.yml

```diff
@@ -25,14 +23,47 @@ featured_item:
         - "6 July 2015"
         - "Brussels"
     title:
-      type: "text"
+      type: "array"
       label: "Title"
-      description: "The title of the page."
-      preview: "Business, Economy, Euro"
+      description: "The title of the page. It must be an array compatible with Link component. If the variant is 'tile', only the label property is required."
+      preview:
+        label: "Business, Economy, Euro"
+        path: "http://example.com"
+    image:
+      type: "array"
+      label: "Image"
+      description: "Image of the featured item."
+      preview:
+        src: "https://placeimg.com/1000/500/arch"
+        alt: "Alternative text for featured item image"
+    tags:
+      type: "array"
+      label: "Tags"
+      description: "List of tags compatible with EC Tag component structure."
+      preview:
+        - href: "http://example.com"
+          label: "Tag 1"
+        - href: "http://example.com"
+          label: "Tag 2"
```

Comment:
This field did not exists in our pattern and shouldn't be added (unless requested by business). Let's leave it out for now.